bin/picca_metal_dmat.py

```diff
@@ -416,6 +422,16 @@ def main(cmdargs):
         for index2, abs_igm2 in enumerate(abs_igm_2[index0:]):
             if index1 == 0 and index2 == 0:
                 continue
+
+            if args.fast_metals:
+                default_si_metals = ['SiIII(1207)','SiII(1190)','SiII(1193)','SiII(1260)']
```

Comment:
this should be defined as a constant at the beginning of the file `DEFAULT_SI_METALS= ['SiIII(1207)','SiII(1190)','SiII(1193)','SiII(1260)']`